### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by mutugi-c**
In this milestone of the portfolio website, the following is implemented in regard to the mobile menu:
- The mobile menu appears when the user clicks (or taps) the hamburger button.
- The mobile menu disappears when the user clicks (or taps) the close (X) button, 
- The mobile menu disappears when the user clicks (or taps) any of the mobile menu options,
- A correct part of the page is displayed when the user clicks (or taps) any of the mobile menu options.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,69 +18,72 @@
...
-            <li class="nav__list-item"><a href="#">Contacts</a></li>
+            <li class="nav__item">
+              <a href="#works" class="nav__link">Portfolio</a>
+            </li>
+            <li class="nav__item">
+              <a href="#about-myself-wrapper" class="nav__link">About</a>
+            </li>
+            <li class="nav__item">
+              <a href="#contacts-container" class="nav__link">Contacts</a>
+            </li>
```
Comment (algerina) [inline index.html:34]: - [ ] You did well working on the mobile menu however you still have to add an (X) button to close the menu as required and as per the Figma design.

Your Project:
review

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @mutugi-c  ,

Good job so far! 

May I highlight the following:
- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## CHANGES REQUIRED ♻️ 

Your project is almost complete you just have some minor changes to do.


- [x] Please check the requirements under the review


 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Comment by mutugi-c** [comment]
@algerina Thanks for pointing that out. I did not notice it vanished when I adjusted the z-index of the nav menu.

**Review by Gambit142 — APPROVED** [review]
Hi @mutugi-c @Alejandroq12 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Used JavaScript to create mobile menu ✔️ 
- Closing icon works ✔️ 
- Links navigate users to respective sites ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**